sdk/src/main/java/com/silanis/esl/sdk/builder/SignerBuilder.java

```diff
@@ -179,7 +180,7 @@ private Signer buildGroupSigner(){
         result.setMessage(message);
         result.setId(id);
         result.setAttachmentRequirements(attachments);
-
+        result.setLocalLanguage(localLanguage);
```

Comment:
Do we need this local language field to use the local language? We set the local language as data in Role.
Why do we need duplicate settings?